spec/requests/demo_mode/sessions_spec.rb

```diff
@@ -131,28 +130,21 @@
 
             expect(DummyUser.last.name).to eq 'New Example Tester'
             expect(response_json['id']).to eq last_session.id
-            expect(response_json['processing']).to be true
-            expect(response_json['username']).to be_nil
-            expect(response_json['password']).to be_nil
+            expect(response_json['processing']).to be false
+            expect(response_json['username']).to eq "New Example Tester"
           end
         end
       end
 
-      context 'with test adapter' do
-        around do |example|
-          queue_adapter_was = ActiveJob::Base.queue_adapter
-          ActiveJob::Base.queue_adapter = :test
-          example.run
-        ensure
-          ActiveJob::Base.queue_adapter = queue_adapter_was
+      context 'with inline adapter' do
+        before do
+          ActiveJob::Base.queue_adapter = :inline
         end
 
         it 'creates a session synchronously and returns default metadata' do
-          perform_enqueued_jobs do
-            post '/ohno/sessions', params: {
-              session: { persona_name: 'the_everyperson' },
-            }.to_json, headers: request_headers
-          end
+          post '/ohno/sessions', params: {
+            session: { persona_name: 'the_everyperson' },
+          }.to_json, headers: request_headers
```

Comment:
We don't need the `perform_enqueued_jobs` since we are globally changing the adapter above.